recommend_from_many.py

```diff
@@ -0,0 +1,13 @@
+# import numpy as np
```

Comment:
Either fix the script or delete it. No leaving commented out code around!

Reply:
Chatao hein

Reply:
Deleted